ouroboros-network/src/Ouroboros/Network/Protocol/BlockFetch/Direct.hs

```diff
@@ -0,0 +1,63 @@
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TupleSections #-}
+
+module Ouroboros.Network.Protocol.BlockFetch.Direct where
+
+import Ouroboros.Network.Protocol.BlockFetch.Client
+import Ouroboros.Network.Protocol.BlockFetch.Server
+
+-- | Directly connect server and client adts of @'BlockFetchClientProtocol'@.
+--
+directBlockFetchClient
```

Comment:
This, as well as `directBlockFetchServer`, should be renamed, because both of these are clients and servers. This just highlights the weirdness of the definitions: each one does only half of the job.

Reply:
I chose the names with this key (I'm likely doing a calque here): one part of the protocol is called `Client` and it has it's client and the orhter `Server`; I will rename them as `BlockReq` and `BlockFetch`, e.g. `BlockReqProtocol`, `directBlockReq`, and so on ..., `BlockFetchProtocol`, `directoBlockFetch`, ...